lib/index.js

```diff
@@ -13,6 +13,23 @@ export default class WebComponent extends ExtendableHTMLElement {
     }
   }
 
+  // Parse number attributes. Examples:
+  // null      -> null
+  // ""        -> null
+  // "asd"     -> null
+  // "123asd"  -> null
+  // "123"     -> 123
+  // "123.45"  -> 123.45
+  getNumberAttribute(attrName) {
+    const attrVal = this.getAttribute(attrName);
+    if (!attrVal) {
+      return null;
+    } else {
+      const num = Number(attrVal);
```

Comment:
Any particular motivations for using this method to parse, as opposed to say `parseInt`?

Reply:
`parseInt('123asd')` -> `123` which IMO opinion is undesired. Also, `Number` works for both int and float.

Reply:
fair enough 🌮 🌮